### PR TITLE
Fixes aimed projectile spell for the third time

### PR DESCRIPTION
**Pull request by silicons**
ugh
it wasn't properly burst-firing.

### CODE REVIEW

**Comment by kevinz000** [comment]
>doesnt compile

NOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

File: code/modules/spells/spell_types/aimed.dm [hunk 1]
```diff
@@ -36,7 +36,10 @@
 /obj/effect/proc_holder/spell/aimed/InterceptClickOn(mob/living/caller, params, atom/target)
 	if(..())
 		return FALSE
-	if(!cast_check(0, ranged_ability_user))
+	var/ignore = TRUE
+	if(current_amount <= 0)
+		ignore = FALSE
```
Comment (AnturK) [inline code/modules/spells/spell_types/aimed.dm]: You could just pass the value in if you only need it once ?

Comment (optimumtact) [inline code/modules/spells/spell_types/aimed.dm]: or just do
var/ignore = (current_amount <= 0)

Comment (optimumtact) [inline code/modules/spells/spell_types/aimed.dm]: @kevinz000 hello

Comment (kevinz000) [inline code/modules/spells/spell_types/aimed.dm]: Shit woops please give me about twelve hours

**Comment by kevinz000** [comment]
kk @optimumtact

**Pull request merged**